smart_card_connector_app/build/human_readable_supported_readers.txt

```diff
@@ -0,0 +1,393 @@
+ACS ACR 38U-CCID
```

Comment:
Let's also change the Python script to print a clarification like:

`# NOTE: Auto-generated by create_human_readable_supported_readers.py.`
`# DO NOT MODIFY MANUALLY!`